README.md

```diff
@@ -115,7 +115,7 @@ def blog_fields():
 
 main = zoom.collect.Collection(
     blog_fields,
-    url=zoom.system.app.url,
+    url='/blog',
```

Comment:
Hey @zodman - thanks for the PR.  What this code is trying to do is avoid hard coding the app URL path.  We do this because we're not sure when we're writing an app how that app will be installed on the site.  It may be under a different path, even though it's technically in a 'blog' repo.

I am assuming you've done this because you've tried the framework and experienced an error.  The standard way to do this now is to call the zoom.get_app() function to get the app.  And then just use it's name, so it could be rewrittent to be `url=zoom.get_app().url`.